### PR TITLE
add setting to config-> disable butterflies spawn

### PR DESCRIPTION
**Pull request by Quetz4l**
+ default "disable butterflies spawn" is false

Butterflies fly behind the loaded chunk and pile up by the hundreds, the chunk crashes when it is loaded
And despawn all butterflies in loaded chunks

### CODE REVIEW

**Comment by Zereff06** [comment]
There are similar mechanics

**Comment by Dream-Master** [comment]
Why we not limited the spawn rate and purge butterflies if there to mannies like we did with special mobs ? We have the source from forestry.

File: src/main/java/forestry/core/config/Config.java [hunk 1]
```diff
@@ -31,6 +31,7 @@
 
 public class Config {
 
+    public static Boolean DISABLE_BUTTERFLY;
```
Comment (eigenraven) [inline src/main/java/forestry/core/config/Config.java:34]: Please follow the style of the rest of the code - camelCase config variable name, use a `boolean` not a boxed `Boolean`, put this field below the categories together with other configs.

**Comment by Dream-Master** [comment]
@Zereff06

**Review by eigenraven — APPROVED** [review]

**Pull request merged**